src/App.js

```diff
@@ -1,17 +1,62 @@
-import './App.css';
+import { useEffect, useState } from "react";
+import ListOfClasses from "./components/ListOfClasses";
+import ListOfMembers from "./components/ListOfMembers";
+import ListOfTrainers from "./components/ListOfTrainers";
 
+export default function App() {
+  const [trainers, setTrainers] = useState([])
+  const [classes, setClasses] = useState([])
+  const [members, setMembers] = useState([])
 
+  console.log("Inside App State: ", {
+    trainers,
+    classes,
+    members
+  })
+  useEffect (() => {
+    fetch("http://localhost:3030/classes")
+    .then((res) => res.json())
+    .then((classData) => {
+      setClasses(classData.data);
+      console.log("Inside Classes Get Fetch: ", classData)
+    });
+  }, [])
 
-function App() {
+    useEffect (() => {
```

Comment:
Using multiple `useEffect`'s is fine, but to keep code clean you might want to consider an approach like this in future:

```
function getClasses() {
    fetch("http://localhost:3030/classes")
        .then((res) => res.json())
        .then((classData) => {
          setClasses(classData.data);
          console.log("Inside Classes Get Fetch: ", classData)
        });
}

function getMembers() {
    fetch(...) etc.
}

useEffect(() => {
    getClasses();
    getMembers();
}, [])

```

Reply:
Will follow your suggestion. It looks better